scripts/wrappers/addons.py

```diff
@@ -4,11 +4,14 @@
 import shutil
 import subprocess
 import sys
+import jsonschema
```

Comment:
I hope it's ok to add a new dependency :/ otherwise, I could try to check the contents of `addons.yaml` "manually"